nodes/logger/logger.html

```diff
@@ -1,165 +1,182 @@
+      <dd>see prefix</dd>
+    </a>
+
+    <dt class='optional'>identifier<span class='property-type'>string</span></dt>
+    <a href='#identifier-description'>
+      <dd>see identifier</dd>
+    </a>
+
+    <dt class='optional'>extension<span class='property-type'>string</span></dt>
+    <dd>For extension with msg object the dropdown must be set to `string`.</dd>
+
+  </dl>
+  <h3>Outputs</h3>
+  <ol class='node-ports'>
+    <li>Standard output
+      <dl class='message-properties'>
+        <dt>payload <span class='property-type'>array|object</span></dt>
+        <dd>Logged data is saved in <code>payload</code></dd>
+      </dl>
+    </li>
+  </ol>
+  <h3>Details</h3>
+  <h4>Config</h4>
+  <p>In the Config, logs are stored under a defined path in an archive folder and deleted when they reach a specific
+    age.</p>
+  <h4 class='fa fa-random'> Write To</h4>
+  <p>The logged input can be written into the <code>debug window</code> as well as into the <code>system
+    console</code>.</p>
+  <h4>What should be logged</h4>
+  <p>Either the <code>payload</code> can be logged, or it is defined via JSONata what is to be logged.</p>
+  <h4><b>Variable Selection</b></h4>
+  <p>The logged data is stored in a defined file in a specific folder. The structure looks like this:
+    <code>PAYLOAD_LOGS_PATH/{folder}/{prefix}_{identifier}_msgid.{extension}</code></p>
+  <h4>PAYLOAD_LOGS_PATH</h4>
+  <p>This is an Environment variable for the log folder. It can be defined in the <code>settings.js</code> file. If
+    this variable is undefined the default value ist <code>./logs`</code></p>
+  <h4 id='folder-description'>Folder</h4>
+  <p>The logged data is stored in this <code>folder</code>.</p>
+  <h4 id='prefix-description'>Prefix</h4>
+  <p>The <code>prefix</code> is the first part of the file name in which the logged data is stored.</p>
+  <h4 id='identifier-description'>Identifier</h4>
+  <p>The <code>identifier</code> is the second part of the file name. It is defined via JSONata. It does not have to
+    be
+    defined.</p>
+  <h4 id='extension-description'>File extension</h4>
+  <p>The file <code>extension</code> can be selected via a dropdown menu. You can choose between json, html, xml, txt
+    and a
+    self-selectable extension.</p>
 </script>
 
 <script type='text/javascript' xmlns='http://www.w3.org/1999/html'>
   RED.nodes.registerType('Logger', {
-    category: 'Autexis',
+    category: 'autexis',
     color: '#5274cb',
     defaults: {
-      name: {value: ''},
+      name: { value: '' },
       logConfig: {
         value: '',
         type: 'LogConfig',
       },
-      debugWindow: {value: false},
-      console: {value: false},
-      logType: {value: ''},
-      logContent: {value: ''},
-      folder: {value: ''},
-      prefix: {value: ''},
-      identifier: {value: ''},
-      extension: {value: ''},
-      extensionHidden: {value: ''},
+      appendingToFile: {value:'true'},
+      appendNewline: {value:false},
```

Comment:
Gibt es einen bestimmten Grund für true als default-Wert?
Und wieso als String?
```suggestion
      appendingToFile: { value: false },
      appendNewline: { value: false },
```